geocatbridge/servers/views/geonetwork.ui

```diff
@@ -115,6 +115,57 @@
           </property>
          </widget>
         </item>
+        <item row="4" column="0">
+         <widget class="QLabel" name="label_33">
+          <property name="text">
+           <string>Group</string>
```

Comment:
`Group ID` would be better here, I think. So it's clear that it should be a (numeric) identifier.